### PR TITLE
What: extend #622 PR to move images for both papers

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
Why:

* @av pointed out best practice to keep images in a separate folder

This change addresses the need by:

* updated all image links for all papers and translations.

### CODE REVIEW

File: supply-chain-security/supply-chain-security-paper/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Software Supply Chain Security Paper
+SSoftware Supply Chain Security Paper
```
Comment (lumjjb) [inline supply-chain-security/supply-chain-security-paper/README.md:1]: nit: typo

File: supply-chain-security/supply-chain-security-paper/README.md [hunk 2]
```diff
@@ -58,4 +58,5 @@ approval on the PR.  At which point the markdown state will be changed to
 
 
 Links:
-* [Managed version in  Markdown](https://github.com/cncf/sig-security/blob/master/supply-chain-security/supply-chain-security-whitepaper/software-supply-chain-security-whitepaper.md)
+* [Managed version in  Markdown](https://github.com/cncf/tag-security/blob/master/supply-chain-security/supply-chain-security-paper/software-supply-chain-security-whitepaper.md)
```
Comment (lumjjb) [inline supply-chain-security/supply-chain-security-paper/README.md:61]: I think this is now `sscsp.md`?

Comment (TheFoxAtWork) [reply supply-chain-security/supply-chain-security-paper/README.md:61]:
```suggestion
* [Managed version in  Markdown](https://github.com/cncf/tag-security/blob/master/supply-chain-security/supply-chain-security-paper/sscsp.md)
```

**Review by lumjjb — APPROVED** [review]
LGTM

**Pull request merged**